can/map/map.js

```diff
@@ -296,11 +296,15 @@ var canMapBehavior = connect.behavior("can/map",function(baseConnection){
 		 */
 		updatedList: function(list, listData, set){
 			queues.batch.start();
-			queues.mutateQueue.enqueue(baseConnection.updatedList, this, arguments,{
-				//!steal-remove-start
-				reasonLog: ["set", set,"list", list,"updated with", listData]
-				//!steal-remove-end
-			});
+			if(process.env.NODE_ENV !== 'production') {
+				queues.mutateQueue.enqueue(baseConnection.updatedList, this, arguments,{
```

Comment:
This function still needs to be called in production. Only the reasonLog part needs to be omitted. So you'll want to refactor this code, something like:

```js
var enqueueOptions = {};

if(process.env.NODE_ENV !== 'production') {
  enqueueOptions = {
    reasonLog: ["set", set,"list", list,"updated with", listData]
  };
}

queues.mutateQueue.enqueue(baseConnection.updatedList, this, arguments, enqueueOptions);
```

Reply:
Thank you I commited the changes!